keyboards/yd68/xm04/main.c

```diff
@@ -0,0 +1,99 @@
+#include <avr/io.h>
+#include <avr/power.h>
+#include <avr/wdt.h>
+#include "lufa.h"
+#include "print.h"
+#include "sendchar.h"
+#include "xm04.h"
+#include "xm04_task.h"
+//#include "serial.h"
+#include "keyboard.h"
+#include "keycode.h"
+#include "action.h"
+#include "action_util.h"
+#include "wait.h"
+#include "suart.h"
+#include "suspend.h"
+extern bool dozing;
+extern bool sleeping;
+extern bool force_usb;
+
+static int8_t sendchar_func(uint8_t c)
+{
+    //xmit(c);        // SUART
+    sendchar(c);    // LUFA
+    return 0;
+}
+
+static void SetupHardware(void)
+{
+    /* Disable watchdog if enabled by bootloader/fuses */
+    MCUSR &= ~(1 << WDRF);
+    wdt_disable();
+
+    /* Disable clock division */
+    clock_prescale_set(clock_div_1);
+
+    // Leonardo needs. Without this USB device is not recognized.
+    USB_Disable();
+
+    USB_Init();
+
+    // for Console_Task
+    USB_Device_EnableSOFEvents();
+    print_set_sendchar(sendchar_func);
```

Comment:
I think this should be `print_set_sendchar(sendchar_func( ));`

Reply:
It does take a function pointer, so print_set_sendchar(sendchar_func); is correct. keyboards/hhkb/rn42/main.c uses the same form.